data.sql

```diff
@@ -166,3 +166,91 @@ UPDATE animals SET owner_id = 3 WHERE name IN  ('Devimon' ,'Plantmon');
+     (SELECT id FROM species WHERE name='Digimon')),
+    ((SELECT id FROM vets WHERE name='Stephanie Mendez'),
+     (SELECT id FROM species WHERE name='Pokemon')),
+    ((SELECT id FROM vets WHERE name='Jack Harkness'),
+     (SELECT id FROM species WHERE name='Digimon'));    
+END;
+
+BEGIN;
+INSERT INTO visits (vet_id, animal_id, visited) 
+                    VALUES ((SELECT id FROM vets WHERE name='William Tatcher'),
+    (SELECT id FROM animals WHERE name='Agumon'),
+    '2020-05-24'),
+    ((SELECT id FROM vets WHERE name='Stephanie Mendez'),
+    (SELECT id FROM animals WHERE name='Agumon'),
+    '2020-04-22'),
+    ((SELECT id FROM vets WHERE name='Jack Harkness'),
+    (SELECT id FROM animals WHERE name='Gabumon'),
+    '2021-02-02'),
+    ((SELECT id FROM vets WHERE name='Maisy Smith'),
+    (SELECT id FROM animals WHERE name='Pikachu'),
+    '2020-01-05'),
+    ((SELECT id FROM vets WHERE name='Maisy Smith'),
+    (SELECT id FROM animals WHERE name='Pikachu'),
+    '2020-03-08'),
+    ((SELECT id FROM vets WHERE name='Maisy Smith'),
+    (SELECT id FROM animals WHERE name='Pikachu'),
+    '2020-05-14'),
+    ((SELECT id FROM vets WHERE name='Stephanie Mendez'),
+    (SELECT id FROM animals WHERE name='Devimon'),
+    '2021-05-04'),
+    ((SELECT id FROM vets WHERE name='Jack Harkness'),
+    (SELECT id FROM animals WHERE name='Charmander'),
+    '2021-02-24'),
+    ((SELECT id FROM vets WHERE name='Maisy Smith'),
+    (SELECT id FROM animals WHERE name='Plantmon'),
+    '2019-12-21'),
+    ((SELECT id FROM vets WHERE name='William Tatcher'),
+    (SELECT id FROM animals WHERE name='Plantmon'),
+    '2020-08-10'),
+    ((SELECT id FROM vets WHERE name='Maisy Smith'),
+    (SELECT id FROM animals WHERE name='Plantmon'),
+    '2021-04-07'),
+    ((SELECT id FROM vets WHERE name='Stephanie Mendez'),
+    (SELECT id FROM animals WHERE name='Squirtle'),
+    '2019-09-29'),
+    ((SELECT id FROM vets WHERE name='Jack Harkness'),
+    (SELECT id FROM animals WHERE name='Angemon'),
+    '2020-10-03'),
+    ((SELECT id FROM vets WHERE name='Jack Harkness'),
+    (SELECT id FROM animals WHERE name='Angemon'),
+    '2020-11-04'),
+    ((SELECT id FROM vets WHERE name='Maisy Smith'),
+    (SELECT id FROM animals WHERE name='Boarmon'),
+    '2019-01-24'),
+    ((SELECT id FROM vets WHERE name='Maisy Smith'),
+    (SELECT id FROM animals WHERE name='Boarmon'),
+    '2019-05-15'),
+    ((SELECT id FROM vets WHERE name='Maisy Smith'),
+    (SELECT id FROM animals WHERE name='Boarmon'),
+    '2020-02-27'),
+    ((SELECT id FROM vets WHERE name='Maisy Smith'),
+    (SELECT id FROM animals WHERE name='Boarmon'),
+    '-2020-08-03'),
+    ((SELECT id FROM vets WHERE name='Stephanie Mendez'),
+    (SELECT id FROM animals WHERE name='Blossom'),
+    '2020-05-24'),
+    ((SELECT id FROM vets WHERE name='William Tatcher'),
+    (SELECT id FROM animals WHERE name='Blossom'),
+    '2021-01-11');
+END;    
```

Comment:
- [ ] You can refer to the above way to insert the data into visits table, below is sample for one of the insertion data.

 `INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24');`